### ISSUE TITLE
Request: YOOtheme

### ISSUE THREAD

**Issue by CodeShakingSheep** [simple-icons/simple-icons#11103]
### Brand Name

YOOtheme

### Website

https://yootheme.com/

### Popularity Metric

The Similarweb rank is 314,653. See https://www.similarweb.com/website/yootheme.com

### Official Resources for Icon and Color

- SVG: https://yootheme.com/site/images/yootheme-logo.svg
- Color: `#4C89F2` 

### Additional Comments

Edit: Does this require to ask for permission @uncenter? From https://yootheme.com/terms

> All materials related to our website or Services, the Services themselves, and all Intellectual Property Rights contained therein or related thereto, including text, graphics, photos, videos, trademarks, logos, sounds, music, user interfaces and source code is owned by or licensed to us and is protected by copyright, trademark and other E.U./European and foreign laws. These Terms of Service do not grant you any right, title or license to copy, reproduce, sell, or otherwise distribute any part of the intellectual property.

**Comment by uncenter** [comment]
Yeah looks like it. Do you mind reaching out? Thank you for checking!

**Comment by CodeShakingSheep** [comment]
> Yeah looks like it. Do you mind reaching out? Thank you for checking!

Ok, I will reach out to them. Do you think it might make sense to include a check if requesting permission is required in the issue template?

**Comment by uncenter** [comment]
> > Yeah looks like it. Do you mind reaching out? Thank you for checking!
> 
> Ok, I will reach out to them. Do you think it might make sense to include a check if requesting permission is required in the issue template?

Thanks for the suggestion! I'll bring this up in our maintainers chat and we'll see if this makes sense.

**Comment by uncenter** [comment]
> Do you think it might make sense to include a check if requesting permission is required in the issue template?

I think the consensus is that having maintainers do the research separately from the requester is better, as generally requesters won't know or care where to look to find the critical legal documents. Additionally, seeing as this is an important consideration for adding new icons, maintainers will have to double check anyway - so it places unnecessary burden and responsibility upon the requester. Thanks for raising the question though!

**Comment by PeterShaggyNoble** [comment]
See #13453
